tests/next_tests/integration_tests/feature_tests/ffront_tests/ffront_test_utils.py

```diff
@@ -32,20 +41,33 @@ def no_backend(program: itir.FencilDefinition, *args: Any, **kwargs: Any) -> Non
     raise ValueError("No backend selected! Backend selection is mandatory in tests.")
 
 
+OPTIONAL_PROCESSORS = []
+if dace_iterator:
+    OPTIONAL_PROCESSORS.append(dace_iterator.run_dace_iterator)
+
+
 @pytest.fixture(
     params=[
         roundtrip.executor,
         gtfn_cpu.run_gtfn,
         gtfn_cpu.run_gtfn_imperative,
         gtfn_cpu.run_gtfn_with_temporaries,
-        dace_iterator.run_dace_iterator,
-    ],
+    ]
+    + OPTIONAL_PROCESSORS,
     ids=lambda p: next_tests.get_processor_id(p),
 )
 def fieldview_backend(request):
+    backend = request.param
+    backend_id = next_tests.get_processor_id(backend)
+
+    if skip_cases := next_tests.BACKEND_SKIP_TEST_MATRIX.get(backend_id, []):
+        for marker, skip_mark, msg in skip_cases:
```

Comment:
I realized that the `if` is just redundant here:
```suggestion
    for marker, skip_mark, msg in next_tests.BACKEND_SKIP_TEST_MATRIX.get(backend_id, []):
```

Reply:
Yes, I realised it too.. I thought it was to keep the line shorter.